File: inui/foundation/blocks/clipped_circle_graph/clipped_circle_graph.py
from inui.elements import *
from inui.svg import *
Html(

    data = (
        Body(

            data = (
                Div(
                    classs='''clipped-circle-graph''',
                    data_clipped_circle_graph='',
                    data_percent='''50''',
                    data = (
                        Div(
                            classs='''clipped-circle-graph-progress''',
                            data = (
                                Div(
                                    classs='''clipped-circle-graph-progress-fill''',
                                ), )
                        ), 
                        Div(
                            classs='''clipped-circle-graph-percents''',
                            data = (
                                Div(
                                    classs='''clipped-circle-graph-percents-wrapper''',
                                    data = (
                                        Span(
                                            classs='''clipped-circle-graph-percents-number''',
                                        ), 
                                        Span(
                                            classs='''clipped-circle-graph-percents-units''',
                                            data = ('''of 100''',)
                                        ), )
                                ), )
                        ), )
                ), )
        ), )
)

File: inui/foundation/blocks/clipped_circle_graph/clipped_circle_graph.js
$("[data-clipped-circle-graph]").each(function() {
  var $graph = $(this),
      percent = parseInt($graph.data('percent'), 10),
      deg = 30 + (300*percent)/100;
  if(percent > 50) {
    $graph.addClass('gt-50');
  }
  $graph.find('.clipped-circle-graph-progress-fill').css('transform','rotate('+ deg +'deg)');
  $graph.find('.clipped-circle-graph-percents-number').html(percent+'%');
});
